docs/tech_stack.md

```diff
@@ -32,15 +32,13 @@ hide:
 
 <h4> BeSecure Environment for Blue teams/Teaming </h4>
 
-<p> This is a development environment configured for an open source project, pre-bundled with all the required tools and dependencies that would help the developer to perform their development activities which includes adding new features, bug/vulnerability fixing etc. </p>
-<p>This is main area where the issue/vulnerabilities tracked by the Red team will be addressed. 
+<p> This is an environment configured for an open source project, prebundled with the required security tools and dependencies that would permit the Blue team to validate application of security patches and confirm compliance to all defined security controls. 
  </p>
 
 <h4> BeSecure Environment for Red Teams/Teaming </h4>
 
-<p> This is a security environment configured for an open source project, pre-bundled with all the required tools and dependencies that would help the red team members to perform their security testing activities.</p>
-<p>Team will be identify the bugs/related vulnerabilities using various security tools installed in this environment. 
+<p> This is an environment configured for the Red team, prebundled with the required security tools and dependencies to conduct red teaming activities on an open source project with focus on unearthing vulnerabilities and exploits to compromise the security posture of the project. 
  </p>
 
-
+The Security assessment report of the tracked project will be updated under [BeSLighthouse](https://be-secure.github.io/BeSLighthouse) project.
```

Comment:
Check if its ok to provide a heading here to separate this content from the rest.